src/main/frontend/components/search.cljs

```diff
@@ -211,8 +211,14 @@
                                  :block
                                  block))
 
-                              nil)
-                            (search-handler/clear-search!))
```

Comment:
I think if the user shift-click/enters an item, we do not need to close the auto-complete menu, but allow the user to open multiple views.

Reply:
Indeed, it's much better than the current behaviour.